docs/install/collecting-metrics/README.md

```diff
@@ -0,0 +1,106 @@
+This document describes how to set up the
+[OpenTelemetry Collector](https://opentelemetry.io/docs/collector/about/) to
+receive metrics from the Knative infrastructure components and distribute them
+to Prometheus. [OpenTelemetry](https://opentelemetry.io/) is a CNCF project to
+provide consistent interfaces for instrumenting applications and platforms to
+provide telemetry information for developers and operators. OpenTelemetry allows
+Knative to build provider-agnostic instrumentation into the platform, so that
+it's easy to export logs to multiple monitoring services without needing to
+rebuild or reconfigure the Knative binaries.
+
+## Setting up the collector
+
+The collector provides a long-lived location where various Knative components
+can push metrics (and eventually traces) to be retained and collected by a
+monitoring service. For this example, we'll configure a single collector
+instance using the
+[opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator).
+
+![Diagram of components reporting to collector, which is scrapde by Prometheus](./system-diagram.svg)
+
+<!-- yuml.me UML rendering of:
+[queue-proxy1]->[Collector]
+[queue-proxy2]->[Collector]
+[autoscaler]->[Collector]
+[Collector]<-scrape[Prometheus]
+-->
+
+1. The OpenTelemetry Operator requires [cert-manager](https://cert-manager.io/).
+   Typically, Knative installations will already have cert-manager for auto-TLS,
+   but if not,
+   [install from these instructions](https://cert-manager.io/docs/installation/kubernetes/).
+
+1. The first step is to install the operator:
+
+   ```shell
+   kubectl apply --filename https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
+   ```
+
+1. Next, create a namespace for the collector to run in:
+
+   ```shell
+   kubectl create namespace metrics
+   ```
+
+1. And then create an OpenTelemetryCollector object to store the configuration
+   for the collector:
+
+   ```shell
+   kubectl apply --filename collector.yaml
+   ```
+
+1. Finally, update the `config-observability` ConfigMap in Knative Serving and
+   Eventing
+   ```shell
+   kubectl patch --namespace knative-serving configmap/config-observability \
```

Comment:
From the diagram it looks like only `queue-proxy` and `activator` would be pushing metrics, what about the control-plane controllers? Would they also be pushing metrics or not?

Reply:
Yes, they would too, I can add more elements, thanks for the callout!

Reply:
We should add Eventing objects like Sources which emit metrics.